runtime/contract_update_validation_test.go

```diff
@@ -1404,6 +1504,38 @@ func assertConformanceMismatchError(
 	)
 }
 
+func assertEnumCaseMismatchError(t *testing.T, err error, expectedEnumCase string, foundEnumCase string) {
+	require.Error(t, err)
+	require.IsType(t, &EnumCaseMismatchError{}, err)
+	enumMismatchError := err.(*EnumCaseMismatchError)
+
+	assert.Equal(
+		t,
+		fmt.Sprintf(
+			"mismatching enum case: expected `%s`, found `%s`",
+			expectedEnumCase,
+			foundEnumCase,
+		),
+		enumMismatchError.Error(),
+	)
+}
+
+func assertMissingEnumCasesError(t *testing.T, err error, declName string, expectedCaes int, foundCases int) {
+	require.Error(t, err)
+	require.IsType(t, &MissingEnumCasesError{}, err)
+	extraFieldError := err.(*MissingEnumCasesError)
+	assert.Equal(
+		t,
+		fmt.Sprintf(
+			"missing cases in enum `%s`: expected %d or more, found %d",
+			declName,
+			expectedCaes,
+			foundCases,
+		),
+		extraFieldError.Error(),
+	)
```

Comment:
Maybe instead of comparing error messages literally, just check declName, expectedCases, and foundCases of the error directly, so changing the error message doesn't break the test